pybikes/bysykkelgraphql.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import json
+
+import requests
+
+from . import utils
+from .base import BikeShareStation, BikeShareSystem
+
+
+class BySykkelGraphql(BikeShareSystem):
+
+    authed = False
+
+    meta = {
+        'system': 'BySykkelGraphql',
```

Comment:
This name is only explanatory for the codename of the service being used, so should be something informative that helps correlate systems through different cities. This could be `Bysykkel GraphQL`. It's not really used anywhere, actually.